app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/HedvigBottomSheet.kt

```diff
@@ -101,21 +118,69 @@ private fun <T> InternalHedvigBottomSheet(
     containerColor = bottomSheetColors.bottomSheetBackgroundColor,
     contentColor = bottomSheetColors.contentColor,
     contentPadding = contentPadding,
-    dragHandle = {
-      DragHandle(
-        modifier = Modifier
-          .fillMaxWidth()
-          .wrapContentWidth(Alignment.CenterHorizontally)
-          .padding(top = 8.dp, bottom = 20.dp),
-      )
-    },
+    dragHandle = dragHandle
+      ?: {
+        DragHandle(
+          modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentWidth(Alignment.CenterHorizontally)
+            .padding(top = 8.dp, bottom = 20.dp),
+        )
+      },
   ) {
     Column(modifier = Modifier.verticalScroll(rememberScrollState())) {
       content()
     }
   }
 }
 
+@Composable
+fun CrossSellDrugHandle(
+  contentPadding: PaddingValues,
+  modifier: Modifier = Modifier,
+  text: String? = stringResource(R.string.CROSS_SELL_BANNER_TEXT),
+) {
+  val direction = LocalLayoutDirection.current
+  Box(
+    modifier
+      .fillMaxWidth()
+      .height(40.dp)
+      .layout { measurable, constraints ->
+        val paddingStart = contentPadding.calculateStartPadding(direction).roundToPx()
```

Comment:
Since this is quite an odd thing to see here without context, perhaps a comment might be useful
```suggestion
        // M3 sheet does not allow us to "break out" of the content padding so we do it through a layout
        val paddingStart = contentPadding.calculateStartPadding(direction).roundToPx()
```